### DISCUSSION TITLE
APPINSIGHTS INITIALIZATION

### DISCUSSION THREAD

**Discussion by Ritesh-KPM** [iTwin/community#18] [Q&A]
I'm trying to run queries on iModel Console. I'm continuously getting this error. Can someone please help me understand what is the issue?
![iModel Console Error](https://github.com/iTwin/community/assets/154978090/d983a3ae-565e-42a3-90f4-136e171fcad4)

**Comment by williamkbentley** [comment]
Hello 
It appears as if the AppInsights client is not being initialized for some reason. Can you supply us with some more information so we can debug this?

Does the error also happen when using https://imodelconsole.bentley.com/ directly? Or only in the My iTwins application?
Does the error actually prevent you from running queries?
What browser are you using?
Could there be a browser extension blocking AppInsights init?

**Reply by nimam-bentley** [reply]
Hi @Ritesh-KPM , could you please provide the email you used to sign into the Console?

**Reply by Ritesh-KPM** [reply]
ritesh.chavan.kpm@gmail.com
this is the email id I'm using to log in.

**Reply by Ritesh-KPM** [reply]
Does the error also happen when using https://imodelconsole.bentley.com/ directly? Or only in the My iTwins application? ---> **I did not check iModelConsole website, on My iTwin application i got that error message.**
Does the error actually prevent you from running queries? ---> **YES**
What browser are you using? ---> **Microsoft Edge**
Could there be a browser extension blocking AppInsights init? ---> **I'm not using any extensions on browser.**

**Reply by williamkbentley** [reply]
Interesting, well this is definitely a problem. I am not able to reproduce the same error, but AppInsights failures shouldn't prevent the app from functioning. We will look into it.
